examples/io/open_save/read_raster.py

```diff
@@ -0,0 +1,41 @@
+"""
+Open/save a raster
+==================
+
+This example demonstrates the instantiation of a raster through :class:`~geoutils.Raster` and saving with :func:`~geoutils.Raster.save`.
+"""
+
+# %%
+# We open an example raster. The data is, by default, unloaded.
+import geoutils as gu
+
+rast = gu.Raster(gu.examples.get_path("everest_landsat_b4"))
+rast
+
+# %%
+#
+# .. note::
+#        A raster can also be instantiated with a :class:`rasterio.io.DatasetReader` or a :class:`rasterio.io.MemoryFile`, see :ref:`sphx_glr_io_examples_import_export_import_raster.py`.
+#
+# We can print more info on the raster.
+print(rast.info())
```

Comment:
In the raster and vector examples, I would add a few more lines to introduce the main attributes: self.crs, self.bounds, self.res, self.width etc. 
Many people will be looking for these attributes, which are listed only in the API, and most users won't look at it.

Reply:
Done!